index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome books</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <header class="header-container">
+      <h1 class="logo-text">Awsome Books</h1>
+      <nav class="nav-container">
+        <div class="nav-list">
+          <a href="#" class="nav-link" onclick="showContent('book-container')"
+            >List</a
+          >
+          <a href="#" class="nav-link" onclick="showContent('bookForm')"
+            >Add book</a
+          >
+          <a href="#" class="nav-link" onclick="showContent('contact')"
+            >Contact</a
+          >
+        </div>
+      </nav>
+    </header>
+    <h1>All awesome books</h1>
+    <div class="booklist" id="book-container">
+      <span id="bookList"></span>
+    </div>
+    <hr />
+
+    <form id="bookForm" class="hidden">
+      <h1>Add a new book</h1>
+
+      <input type="text" id="titleInput" placeholder="Title" /> <br />
+      <br />
+      <input type="text" id="authorInput" placeholder="Author" /> <br />
+      <br />
+      <button type="submit" class="add">Add</button>
+    </form>
+
+    <div id="contact" class="contact-container hidden">
+      <h2>Contact Information</h2>
+      <p class="contact-text">
+        Do you have any question or just want to say hello? <br />
+        You can reah out to us at!
+      </p>
+      <ul class="contact-options">
+        <li>
+          Our email
+          <a href="mailto:petermmwovi@gmail.com"
+            >petermmwovi@gmail.com</a
+          >
+        </li>
+        <li>Phone: +25470123456</li>
+        <li>Address: Street no 4 Green town Lahore.</li>
+      </ul>
+    </div>
+
```

Comment:
As part of the project requirements, you are to follow the design framework as close as possible. currently, your design lacks a footer, please I'd suggest you add the footer for your website so it will have a functionality that is closely similar to the design given 😃

![image](https://github.com/peter-mwovi/Awesome_books/assets/20537712/414ea11f-545c-46ae-8c1c-8ccb19b680d5)